guide/miscellaneous/useful-packages.md

```diff
@@ -64,13 +66,13 @@ const messages = await message.channel.awaitMessages(m => m.author.id === messag
 const difference = messages.last().createdTimestamp - messages.first().createdTimestamp;
 const formatted = ms(difference);
 
-message.channel.send(`You sent the two messages ${formatted} apart.`);
+interaction.reply(`You sent the two messages ${formatted} apart.`);
```

Comment:
```suggestion
await interaction.followUp(`You sent the two messages ${formatted} apart.`);
```
This should be `.followUp()` since `.reply()` is already used above.